extensions/shared/external-media/media-browser/index.js

```diff
@@ -37,9 +58,12 @@ function MediaBrowser( props ) {
 		onCopy,
 	} = props;
 	const [ selected, setSelected ] = useState( [] );
+	const [ focused, setFocused ] = useState( -1 );
```

Comment:
Should this be a useRef as well instead of in state?

Reply:
Change of focused value should cause a re-render, no? I don't think that a ref would work in this case.

Reply:
I'm not sure. I'm not sure how a render is defined. It's not re-rendering anything in the HTML 🤷

Reply:
The `MediaItem` component visuals depend on the `focused` state, so the `focused` value change should trigger the `MediaItem` update. I hope that is true 😄

Reply:
Tested this out with a ref instead. It only worked for the load more handler, likely because it was already triggering a rerender.